plugins/jobs/server/job_rest.py

```diff
@@ -33,6 +33,8 @@ def __init__(self):
         self.route('GET', ('all',), self.listAllJobs)
         self.route('GET', (':id',), self.getJob)
         self.route('PUT', (':id',), self.updateJob)
+        self.route('GET', (':id', 'child',), self.getChildJobs)
+        self.route('PUT', (':id', 'parent',), self.setParentJob)
```

Comment:
Let's remove this route for now and implement it in https://github.com/girder/girder/pull/2031 once we have a better idea on how we want to handle access control for an endpoint like this.